data/gegno/gegno vi outfits.txt

```diff
@@ -10,7 +10,7 @@ outfit "Limescale Breach Sealant"
 	"hull repair rate" .3
 	"hull energy" .15
 	"hull heat" .3
-	description `This is the first generation of the Gegno Vi's "breach sealant" outfits, a relatively new technology that allows a damaged Vi ship to seal any larger wounds in its hull over time. This system is very low-tech, relying on small welding arms placed throughout the ship, but the ability to recover mid-flight is a game-changer in Gegno space. Sadly, most Gegno Vi captains don't seem convinced that the tradeoff of firepower is worth the change.`
+	description `This is the first generation of the Gegno Vi's "breach sealant" outfits, a relatively new technology that allows a damaged Vi ship to seal any larger wounds in its hull over time. This system is very low-tech, relying on small welding arms placed throughout the ship, but the ability to recover mid-flight is a game-changer in Gegno space. Sadly, most Gegno Vi captains don't seem convinced that the tradeoff in firepower is worth the capacity to repair mid-flight.`
```

Comment:
I feel like "capability" is a better word than "capacity".

Reply:
```suggestion
	description `This is the first generation of the Gegno Vi's "breach sealant" outfits, a relatively new technology that allows a damaged Vi ship to seal any larger wounds in its hull over time. This system is very low-tech, relying on small welding arms placed throughout the ship, but the ability to recover mid-flight is a game-changer in Gegno space. Sadly, most Gegno Vi captains don't seem convinced that the tradeoff in firepower is worth the capability to repair mid-flight.`
```